charts/rstudio-pm/values.yaml

```diff
@@ -129,6 +129,7 @@ pod:
   # -- the [securityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) for the main Package Manager container. Evaluated as a template.
   containerSecurityContext:
     runAsUser: 999
+    fsGroup: 999
```

Comment:
@tylfin I updated to the newest chart and got a warning about an unknown field:
```sh
W0927 16:29:42.405025  521699 warnings.go:70] unknown field "spec.template.spec.containers[0].securityContext.fsGroup"
```

I think `fsGroup` has to go in the `securityContext` block above, not `containerSecurityContext`?

Reply:
Ah okay, let me patch this